api/versions_test.go

```diff
@@ -503,6 +504,19 @@ func TestGetVersionReturnsError(t *testing.T) {
 	})
 }
 
+func storeMockWithLock(expectFirstGetUnlocked bool) (*storetest.StorerMock, *bool) {
```

Comment:
Is this function used?

Reply:
You are right, I copied it from another test but ended up modifying the existing mocks instead - I will remove it.